configure

```diff
@@ -67,6 +67,13 @@ save_configuration
 
 freshen_chef_repo
 
-cd chef_repo
+pushd chef_repo
```

Comment:
I have a slight reservation about using pushd.
When we set up this repo, using bash was acceptable because we were focused on Linux systems. However, I've had to create a separate branch that uses only POSIX shell to support the macOS deployments and I don't think pushd is available in that context.

We could replace `pushd`/`popd` usage by saving the top level directory or just using `..` which gets unwieldy if we do a lot of directory traversal but is "okay" in small amounts.

There are other uses of `pushd`/`popd` in this script so adding this one is not creating a problem and I don't think this is blocking.

Reply:
I see [this](https://github.com/ros-infrastructure/ros-buildfarm-deployment/pull/5/commits/558ff835a87b873c1f960205e3a784774cb4c005) commit on PR #5 , I'm happy to change pushd to something like that if it makes sense.

Reply:
Let's leave it as is since #5 has bitrotted and we'll address it when I refresh #5.